src/Microsoft.SqlTools.ServiceLayer/Connection/Contracts/SecurityTokenRequest.cs

```diff
@@ -0,0 +1,40 @@
+﻿using Microsoft.SqlTools.Hosting.Protocol.Contracts;
+
+namespace Microsoft.SqlTools.ServiceLayer.Connection.Contracts
+{
+    class RequestSecurityTokenParams 
+    {
+        /// <summary>
+        /// Gets or sets authority.
```

Comment:
It would be nice to have these actually say what "authority", "resource" and "scope" are in this context. It's not too hard to figure out given the class name but currently these comments are all but worthless and it would be useful for people that don't necessarily know the finer details of the protocol.

Reply:
Comments updated.